Documentation/FontEditor/UsingFontEditor.md

```diff
@@ -0,0 +1,123 @@
+# Using Font Editor
+
+### Welcome to Font Editor. This is the official app to create and edit bitmap font files in SerenityOS.
+
+![](FontEditor.png)
+
+### Basic Parts
+FontEditor has the following basic parts:
+1. The menu bar at the top
+2. The toolbar
+3. The main workspace
+4. The status bar at the bottom
+
+Presently, you can resize the entire FontEditor form to your liking. You can turn-on or turn-off the visibility of Font Metadata and Unicode Blocks. You can change the glyph editor window zoom factor.  All of these things are currently available in FontEditor automatically.
```

Comment:
You can turn the visibility of Font Metadata and Unicode Blocks on or off.

Reply:
Wow first comment.  :^), thanks much for the input.

Reply:
Updated as such.